core/shared/src/main/scala/fs2/Chunk.scala

```diff
@@ -338,7 +338,41 @@ object Chunk {
       }
 
   /** Creates a chunk backed by a `Seq`. */
-  def seq[O](s: Seq[O]): Chunk[O] = vector(s.toVector)
+  def seq[O](s: Seq[O]): Chunk[O] = s match {
+    case a: collection.mutable.WrappedArray[O] => array(a.array)
```

Comment:
This is the type you get when passing varargs, so things like `Chunk(1, 2, 3)` will be *significantly* faster with this change -- before, the 3-element array was promoted to a `Vector` which is slow and wasteful.